test/test_network_constructors.jl

```diff
@@ -301,12 +301,7 @@ end
 end
 
 @testset "Network Unsupported Power Model Formulations" begin
-    incompat_list = [
-        PM.SDPWRMPowerModel,
-        PM.SparseSDPWRMPowerModel,
-        PM.SOCBFPowerModel,
-        PM.SOCBFConicPowerModel,
-    ]
+    incompat_list = [PM.SOCBFPowerModel, PM.SOCBFConicPowerModel, PM.IVRPowerModel]
```

Comment:
@claytonpbarrows defined this as a const in definitions. Should we import it or leave this definition here?

Reply:
I'll use the const definition.